src/components/App.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState, useEffect } from "react";
+import "../styles/App.css";
+import getForecast from "../requests/getForecast";
+import LocationDetails from "./LocationDetails";
+import ForecastSummaries from "./ForecastSummaries";
+import ForecastDetails from "./ForecastDetails";
+import SearchForm from "./searchForm";
+
+function App() {
+  const [forecasts, setForecasts] = useState([]);
+  const [location, setLocation] = useState({ city: "", country: "" });
+  const [selectedDate, setSelectedDate] = useState(0);
+  const [searchText, setSearchText] = useState("");
+  const [errorMessage, setErrorMessage] = useState("");
+
+  useEffect(() => {
+    getForecast(setSelectedDate, setForecasts, setLocation, setErrorMessage);
```

Comment:
You are not passing in anything as searchText.
UseEffect is used on the first render of the component, we want to see the weather in Manchester when we first load the app. To get the data from the response you are using getForecast function.
This function accepts: searchText, setSelectedDate, setForecasts, setLocation, setErrorMessage as parameters. If you do not pass in searchtext the parameter order gets messed up. 

````
const getForecast = (
  searchText,
  setSelectedDate,
  setForecasts,
  setLocation,
  setErrorMessage
) => {
  let endpoint = "https://cmd-shift-weather-app.onrender.com/forecast";

  if (searchText) {
    endpoint += `?city=${searchText}`;
  }
`````
here you can see the order of the parameters(this is from the getForecast.js file.
The order is first searchText then setSelected date then setForecast then setLocation and the last is setErrorMessage.
but in useEffect you do: 
````
getForecast(setSelectedDate, setForecasts, setLocation, setErrorMessage);
`````
which is FIRST setSelectedDate then setForecasts ... etc.

Which means you are passing in setSelected date AS searchText, and this is why the first call fails, because in the getForecast file on line 13 when you say:
``
```
  if (searchText) {
    endpoint += `?city=${searchText}`;
  }
````
the searchText variable = setSelectedDate function, so your first call goes to:
https://cmd-shift-weather-app.onrender.com/forecast + `?city=${~~searchText~~setSelectedDate()}`

So after all this me rambling away :D 

You have to make sure you are passing in a value for the searchText which on the first render should be "".
(The reason why you should not pass in searchText directly is because searchText is being updated on each key press when you type in a new letter for the city you would like to look for, and this would trigger the useEffect to re-render on each change)